package.json

```diff
@@ -11,18 +11,12 @@
   "main": "dist/index.js",
   "module": "dist/index.mjs",
   "types": "dist/index.d.ts",
-  "exports": {
-    ".": {
-      "require": "./dist/index.js",
-      "import": "./dist/index.mjs",
-      "types": "./dist/index.d.ts"
-    }
-  },
   "files": [
     "dist/**/*",
     "README.md",
     "*.js",
-    "*.d.ts"
+    "*.d.ts",
+    "src/**/*"
```

Comment:
This will [include](https://www.npmjs.com/package/@chromaui/test-archiver/v/0.0.27--canary.24.9cb64d2.0?activeTab=code) the `src/` files for all who use this package, which I don't think we'll want.

Reply:
Ah, yeah, I did this for testing and forgot to remove it again. Thanks for catching this.